SharedSource/Core/Utilities/ArgumentUtility.cs

```diff
@@ -19,7 +19,7 @@
 using System.Diagnostics;
 using System.Reflection;
 using JetBrains.Annotations;
-
+#nullable disable

```

Comment:
This should be `#nullable enable`. Please change and fix errors to allow a second review.